backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHParquetSuite.scala

```diff
@@ -2161,53 +2161,6 @@ class GlutenClickHouseTPCHParquetSuite extends GlutenClickHouseTPCHAbstractSuite
     }
   }
 
-  test("GLUTEN-3534: Fix incorrect logic of judging whether supports pre-project for the shuffle") {
```

Comment:
Let's ignore it first. I think it more like a bug, we must to know the RC.

Reply:
already created the #3731 to trace this issue.

Reply:
OK, We can revert removed test easily, let's go on.